media-video/shotcut/shotcut-18.01.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="ca cs da de el en es fr gd hu it ja nb ne nl oc pl pt_BR pt_PT ru sk sl tr uk zh_CN zh_TW"
+
+inherit eutils qmake-utils
```

Comment:
get rid of `eutills.eclass` here in favor of `desktop.eclass`

Reply:
`l10n` eclass for `PLOCALES` support and I do not see how locales are affect this ebuild at all otherwise